<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account</class>
 <widget class="QMainWindow" name="Account">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>709</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1085</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1085</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: black;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Zmień swoje hasło</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_changePassword">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
background-color: rgb(242, 242, 242);</string>
    </property>
    <property name="text">
     <string>Zamień hasło</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Aktualne hasło</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_currentPassword"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>nowe hasło</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>powtórz hasło</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>431</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Imię</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;</string>
         </property>
         <property name="text">
          <string>abc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Nazwisko</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_surname">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;</string>
         </property>
         <property name="text">
          <string>abc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;
font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>PESEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_id">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;</string>
         </property>
         <property name="text">
          <string>abc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;
font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Adres zamieszkania</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_location">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;</string>
         </property>
         <property name="text">
          <string>abc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;
font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Adres email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_email">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-left: 10px;</string>
         </property>
         <property name="text">
          <string>abc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
